src/dialects/db2/query-generator.js

```diff
@@ -49,8 +45,12 @@ export class Db2QueryGenerator extends AbstractQueryGenerator {
   }
 
   showSchemasQuery() {
-    return 'SELECT SCHEMANAME AS "schema_name" FROM SYSCAT.SCHEMATA WHERE '
-      + '(SCHEMANAME NOT LIKE \'SYS%\') AND SCHEMANAME NOT IN (\'NULLID\', \'SQLJ\', \'ERRORSCHEMA\')';
+    return Utils.toSingleLine(`
```

Comment:
`support.ts` has a function called `minifySql`. I think we could move it to utils instead of `toSingleLine`

Reply:
That would be great! I saw the SQLFragments, but it didn't help here.  This is more a DX problem it solves.  I find well-formatted SQL to be much easier to read and identify problems and performance issues.